Common subdirectories: delegator-0.0.3-orig/delegator and delegator-0.0.3/delegator
Common subdirectories: delegator-0.0.3-orig/delegator.egg-info and delegator-0.0.3/delegator.egg-info
diff -uN delegator-0.0.3-orig/LICENSE delegator-0.0.3/LICENSE
--- delegator-0.0.3-orig/LICENSE	1970-01-01 07:00:00.000000000 +0700
+++ delegator-0.0.3/LICENSE	2013-12-10 23:58:13.000000000 +0700
@@ -0,0 +1,29 @@
+delegate
+========
+
+Python implementation of Ruby's delegate.rb
+
+license
+=======
+
+The MIT License (MIT)
+
+Copyright (c) 2013 Hugo Bastien
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in
+all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+THE SOFTWARE.
diff -uN delegator-0.0.3-orig/README.txt delegator-0.0.3/README.txt
--- delegator-0.0.3-orig/README.txt	2013-12-10 23:58:13.000000000 +0700
+++ delegator-0.0.3/README.txt	2019-03-07 12:28:04.536103159 +0700
@@ -2,28 +2,3 @@
 ========
 
 Python implementation of Ruby's delegate.rb
-
-license
-=======
-
-The MIT License (MIT)
-
-Copyright (c) 2013 Hugo Bastien
-
-Permission is hereby granted, free of charge, to any person obtaining a copy
-of this software and associated documentation files (the "Software"), to deal
-in the Software without restriction, including without limitation the rights
-to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
-copies of the Software, and to permit persons to whom the Software is
-furnished to do so, subject to the following conditions:
-
-The above copyright notice and this permission notice shall be included in
-all copies or substantial portions of the Software.
-
-THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
-IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
-FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
-AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
-LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
-OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
-THE SOFTWARE.
Common subdirectories: delegator-0.0.3-orig/test and delegator-0.0.3/test
